docs/source/contents/4_draft_mapping.rst

```diff
@@ -5194,11 +5194,22 @@ or one of the CC licenses. These values are used to provide users with standard
 status of an item and how or if it can be reused. These values are currently displayed in a facet and are recommended for
 sharing with DPLA.
 
+All creative commons licenses should be content negotiable and valid.  For this to happen, this pattern must be followed:
+
+:code:`http://creativecommons.org/licenses/*/*/rdf`
+
+This means:
+
+* Use :code:`http` instead of :code:`https` as the protocol (for content negotiation and validity)
+* End in :code:`/rdf` (for content negotiation)
+
 Justification
 ^^^^^^^^^^^^^
 
-DPLA maps both CC licenses and Rights Statements to `edm:rights`. So does Samvera. Presently only the Heilman Collection includes
-a CC license.
+DPLA maps both CC licenses and Rights Statements to `edm:rights`. So does Samvera. Presently only the Heilman Collection
```

Comment:
@markpbaggett - I would suggest that we take out the sentence "Presently only the Heilman Collection includes a CC license" before merging. We now have CC licenses in the University of Tennessee Extension Special Circulars and other Project CERES collections as well as quite a few in Galston. This statement therefore is no longer true. I think we'll be using CC a lot more in the future. I'll double check all of our links and create a ticket.